charts/nginx/values.yaml

```diff
@@ -9,7 +9,7 @@ tolerations: []
 images:
   nginx:
     repository: quay.io/astronomer/ap-nginx
-    tag: 1.2.1
+    tag: 1.3.0
```

Comment:
Given that nginx 1.3.0 does not support k8s 1.19, we should also remove k8s 1.19 as part of this PR.

Reply:
seems its only  1.19.0 not the whole series

Reply:
oh, awesome! then we can completely ignore it because we can expect almost nobody to be running exactly 1.19.0.

Reply:
We should probably call this out in our customer facing documentation. Wdyt @danielhoherd, or is that not necessary?